test/model/test/model_rts/main.cpp

```diff
@@ -222,6 +222,7 @@ test_restart_wt1(void)
     /* Create an unnamed checkpoint, crash, and verify. */
```

Comment:
Are we creating an unnamed checkpoint?

Reply:
Thanks for the catch! Fixed in [2522e5d](https://github.com/wiredtiger/wiredtiger/pull/10414/commits/2522e5dd035f5b8a6d34437fcf2f3ce644c135e1).